aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+- When users need to focus on one task at a time.
+- To give information overview with the ability to drill down into specific information.
+- When screen real estate is limited.
+- When designing for many levels of functionality.
+- To divide meaningful content into sections.
+
+## Guidelines:
+- When the user is moving forward step-by-step to access further information.
+- When going from your Account > Dashboard > Service.
+- To allow users to dive deeper into related content.
+- Can be used as an individual element or in groups of related content.
+- Each element should have it’s own single content.
+
+## Drill Down Navigation Examples:
+
+### Card Example
+A visual representation of interactive and organized content.
+
+<Example caption="A diagram illustrating drill down navigation within Cards." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/drill-down-card-example.svg", light: '/templateImages/drill-down-card-example-invert.svg'}} alt="A diagram illustrating drill down navigation within cards." />
+</Example>
```

Comment:
Can we format and indent these for easier readability? Otherwise, it is difficult to parse what is included/left out.

```suggestion
<Example 
  caption="A diagram illustrating drill down navigation within Cards." 
  plain 
  pad="small" 
  previewWidth="large" 
  height={{ min: 'small' }}
>
  <DiagramPreview 
    src={{
      dark: "/templateImages/drill-down-card-example.svg", 
      light: '/templateImages/drill-down-card-example-invert.svg'
    }} 
    alt="A diagram illustrating drill down navigation within cards." 
  />
</Example>
```